src/components/PageContent/PageContent.tsx

```diff
@@ -0,0 +1,43 @@
+import classNames from 'classnames';
+import { NavLink, Outlet } from 'react-router-dom';
+
+export const PageContent = () => {
+  return (
+    <>
+      <nav
+        data-cy="nav"
+        className="navbar is-fixed-top has-shadow"
+        role="navigation"
+        aria-label="main navigation"
+      >
+        <div className="container">
+          <div className="navbar-brand">
+            <NavLink
+              className={({ isActive }) => classNames('navbar-item', {
+                'has-background-grey-lighter': isActive,
+              })}
+              to="/"
+            >
+              Home
+            </NavLink>
+
+            <NavLink
+              className={({ isActive }) => classNames('navbar-item', {
+                'has-background-grey-lighter': isActive,
+              })}
```

Comment:
You have repeatable styles, move this to function(rewatch [this video](https://mate.academy/learn/react-with-typescript/react-router?section=video&videoId=5466))